src/MissionManager/CameraSectionTest.cc

```diff
@@ -90,20 +88,19 @@ void CameraSectionTest::init(void)
                                                                   MAV_FRAME_MISSION,
                                                                   0,                               // camera id = 0, all cameras
                                                                   CameraSection::CameraModeVideo,
-                                                                  NAN, NAN, NAN, NAN, NAN,         // param 3-7 unused
+                                                                  NAN,                             // Audio off/on
+                                                                  NAN, NAN, NAN, NAN,              // param 4-7 reserved
                                                                   true,                            // autocontinue
                                                                   false),                          // isCurrentItem
                                                       this);
     _validTakePhotoItem = new SimpleMissionItem(_offlineVehicle,
                                                 MissionItem(0,
                                                             MAV_CMD_IMAGE_START_CAPTURE,
                                                             MAV_FRAME_MISSION,
-                                                            0,                               // camera id = 0, all cameras
+                                                            0,                              // camera id = 0, all cameras
                                                             0,                              // Interval (none)
                                                             1,                              // Take 1 photo
-                                                            -1,                             // Max horizontal resolution
-                                                            -1,                             // Max vertical resolution
-                                                            0, 0,                           // param 6-7 not used
+                                                            NAN, NAN, NAN, NAN,             // param 4-7 not reserved
```

Comment:
Oops... `"not" reserved` ain't exactly right but I won't restart a build just for that :P